lib/cn/kvset.c

```diff
@@ -240,6 +246,20 @@ kvset_hblk_init(
     if (err)
         return err;
 
+    err = hbr_read_vgroup_cnt(hbd, &nvgroups);
+    if (err)
```

Comment:
I try to use ev() only in cases that won't cause the program to exit, otherwise they just bloat the code.  Although they do contain an unlikely() which helps the compiler generate better code flow (i.e., improves the density of hot code and lowers i-cache misses).

Reply:
There is basically 0 guidance on how to use it, so I have no clue what I am doing. I just throw it willy-nilly at this point